snippets/typescript-snippets.cson

```diff
@@ -18,4 +18,36 @@
              * $1
              */
         """
- 
+    'exportinterface':
+            'prefix': 'exi'
+            'body': """
+                export interface ${1:IName}{${2:value:number;}}
+            """
+    'exportclass':
+        'prefix': 'exc'
+        'body': """
+            export class ${1:Name}{
+            this._init:boolean;
+            constructor(public idx:number, init:boolean){
+                  _init=init;
+                  }
+            memberFunction(){
+
+            }
+          }
+        """
```

Comment:
bad indentation and not enough stops (only $1). No for now: 

![image](https://cloud.githubusercontent.com/assets/874898/6522158/6a81f958-c435-11e4-8040-4126fafe4f38.png)